src/System.IO.IsolatedStorage/src/System/IO/IsolatedStorage/IsolatedStorage.cs

```diff
@@ -108,7 +108,7 @@ internal set
 
         public IsolatedStorageScope Scope
         {
-            get; protected set;
+            get; private set;
```

Comment:
I presume this works. If this doesn't work you can use internal.

Reply:
this works